sqrl-tools/sqrl-run/src/main/java/com/datasqrl/DatasqrlRun.java

```diff
@@ -40,26 +43,39 @@
 import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
 import org.apache.flink.table.api.internal.TableEnvironmentImpl;
 import org.apache.flink.table.operations.StatementSetOperation;
-import org.apache.kafka.streams.integration.utils.EmbeddedKafkaCluster;
-import org.testcontainers.containers.PostgreSQLContainer;
-import org.testcontainers.utility.DockerImageName;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import org.apache.kafka.clients.admin.NewTopic;
 
 @Slf4j
 public class DatasqrlRun {
```

Comment:
This is cleaner if we remove all deps in sqrl-run and build a jar just for this for docker. It resolves some messiness in duplicate dependencies.